src/Umbraco.Core/Logging/SerilogExtensions/Log4NetLevelMapperEnricher.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Serilog.Core;
+using Serilog.Events;
+
+namespace Umbraco.Core.Logging.SerilogExtensions
+{
+    /// <summary>
+    /// This is used to create a new property in Logs called 'Log4NetLevel'
+    /// So that we can map Serilog levels to Log4Net levels - so log files stay consistent
+    /// </summary>
+    public class Log4NetLevelMapperEnricher : ILogEventEnricher
+    {
+        public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            var log4NetLevel = string.Empty;
+
+            switch (logEvent.Level)
+            {
+                case LogEventLevel.Debug:
+                    log4NetLevel = "DEBUG";
+                    break;
+
+                case LogEventLevel.Error:
+                    log4NetLevel = "ERROR";
+                    break;
+
+                case LogEventLevel.Fatal:
+                    log4NetLevel = "FATAL";
+                    break;
+
+                case LogEventLevel.Information:
+                    log4NetLevel = "INFO ";
+                    break;
+
+                case LogEventLevel.Verbose:
+                    log4NetLevel = "ALL  ";
+                    break;
+
+                case LogEventLevel.Warning:
+                    log4NetLevel = "WARN ";
+                    break;
+            }
```

Comment:
@warrenbuckley should the whitespace in `INFO`, `ALL` and `WARN` be removed?

Reply:
I'm guessing this is a hack @warrenbuckley did to ensure the log files stay consistent. Serilog most likely doesn't provide the same padding on the log level string as log4net did when outputting a text file?

Reply:
There's however a function for this that would make it more clear that you're intentionally wanting to do this ... 😄 also ensures noone cleans this up by accident.
https://docs.microsoft.com/en-us/dotnet/standard/base-types/padding

Reply:
@clausjensen @warrenbuckley ooh, I see 💩🙈

Reply:
Will push that change up at lunchtime @clausjensen 👍